components/navigation/arrowLink.js

```diff
@@ -42,6 +43,7 @@ const ArrowLink = ({ children, link, type, content }) => {
               ${styles.Link}
               ${styles.NextLink}
             `}
+            target={target ? target : "_self"}
```

Comment:
SVGs don't need the `target` attribute

Reply:
Yup, I didn't use it for the SVGs. The "Next: Read More ->" (see screenshot) use `arrowLink.js`. Currently, clicking them opens the links in the same tab. I add `target` to `arrowLink.js` and use that in news entry to open the blog posts in a new tab.

![image](https://user-images.githubusercontent.com/20672874/188856983-7ddb69ae-1ac7-41f0-8846-93e8a573cdbc.png)